Task1/sortedList.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+typedef struct Node
+{
+    int value;
+    struct Node* nextNodePtr;
+} Node;
+
+typedef struct List
+{
+    Node* firstNodePtr;
+} List;
```

Comment:
```suggestion
typedef struct List List;
```
Список должен быть абстрактным типом данных. Стало быть внутренне представление должно быть скрыто в .c-файле. А в .h-файле тип должен быть описан без реализации.